pkg/hubble/parser/threefour/parser.go

```diff
@@ -147,13 +147,22 @@ func (p *Parser) Decode(data []byte, decoded *pb.Flow) error {
 	if len(data) < packetOffset {
 		return fmt.Errorf("not enough bytes to decode %d", data)
 	}
+	data = data[packetOffset:]
```

Comment:
this is not great since we now completely lose the header for the rest of the function. it's currently not used i guess so perhaps not a big deal

Reply:
Agree, will drop this and change the check below to `if len(data[packetOffset:) == 0` as per https://github.com/cilium/cilium/pull/14448#discussion_r545967507